master-firmware/src/manipulator/manipulator.h

```diff
@@ -36,7 +36,10 @@ class Manipulator {
         /* MANIPULATOR_DEPLOY_FULLY */ {{1.2705, 1.5002, 1.4614}},
         /* MANIPULATOR_LIFT_HORZ */ {{1.3344, 1.1287, 0.0}},
         /* MANIPULATOR_PICK_HORZ */ {{0.9956, 0.5278, 0.0}},
-    };
+        /* MANIPULATOR_PICK_GOLDONIUM */ {{0.935233, 2.005595, 1.570796}},
+        /* MANIPULATOR_PICK_GOLDONIUM_2 */ {{0.8, 3, 1.5908}},
```

Comment:
What is the difference between pick_goldo and pick_goldo_2 ? Maybe find a better name ?

Reply:
Should be `LIFT_GOLDONIUM`